src/package.xml

```diff
@@ -845,6 +844,7 @@
         <members>Address__c.Congressional_District__c</members>
         <members>Address__c.County_Name__c</members>
         <members>Address__c.Default_Address__c</members>
```

Comment:
@searheart The only changes that should be in this file are the 4 new Address__c fields.